src/main/java/com/easypost/model/Shipment.java

```diff
@@ -1035,8 +1033,22 @@ public Shipment insure(final String apiKey) throws EasyPostException {
      * @param deliveryAccuracy Delivery days accuracy restriction to use when filtering.
      * @return lowest Smartrate object
      * @throws EasyPostException when the request fails.
+     * @deprecated use {@link #lowestSmartRate(int, SmartrateAccuracy)} instead.
```

Comment:
Issue: I believe this is missing a param?

Reply:
No, this redirects users to use `lowestSmartRate` on line 1051.